_posts/patterns/2015-12-11-tables.md

```diff
@@ -0,0 +1,257 @@
+
+info:           Tables are used for displaying information in tabular format, where is useful to be able to examine and compare data or information in columns and rows. edX offers several table styles to provide ways to display most tabular data, including an example that contains the full set of supported semantic elements.
+
+---
+
+<h3 class="hd-6 example-set-hd">Basic</h3>
+<div class="example-set">
+  <table class="table">
+    <tr>
+      <th>First</th>
+      <th>Last</th>
+      <th>Lineage</th>
+    </tr>
+    <tr>
+      <td>Aragorn II</td>
+      <td>Elessar</td>
+      <td>Son of Arathorn</td>
+    </tr>
+    <tr>
+      <td>Bilbo</td>
+      <td>Baggins</td>
+      <td>Bag End</td>
+    </tr>
+    <tr>
+      <td>Meriadoc</td>
+      <td>Brandybuck</td>
+      <td>Buckland</td>
+    </tr>
+    <tr>
+      <td>Treebeard</td>
+      <td>Fangorn</td>
+      <td>Fangorn Forest</td>
+    </tr>
+  </table>
+</div>
+
+<h3 class="hd-6 example-set-hd">Simplified and compact</h3>
+<div class="example-set">
+  <table class="table table-simplified table-cozy">
+    <tr>
+      <th>First</th>
+      <th>Last</th>
+      <th>Lineage</th>
+    </tr>
+    <tr>
+      <td>Aragorn II</td>
+      <td>Elessar</td>
+      <td>Son of Arathorn</td>
+    </tr>
+    <tr>
+      <td>Bilbo</td>
+      <td>Baggins</td>
+      <td>Bag End</td>
+    </tr>
+    <tr>
+      <td>Meriadoc</td>
+      <td>Brandybuck</td>
+      <td>Buckland</td>
+    </tr>
+    <tr>
+      <td>Treebeard</td>
+      <td>Fangorn</td>
+      <td>Fangorn Forest</td>
+    </tr>
+  </table>
+</div>
+
+<h3 class="hd-6 example-set-hd">Full Semantic Set</h3>
+<div class="example-set">
+  <div class="wrapper-table-scrollable">
+    <table class="table">
+      <caption>Some fantastical towns and their key characteristics</caption>
+      <colgroup class="colgroup">
+        <col class="col col1">
+      </colgroup>
+      <colgroup class="colgroup">
+        <col class="col col2">
+        <col class="col col3 is-highlighted">
+        <col class="col col4">
+        <col class="col col5">
+      </colgroup>
+      <colgroup class="colgroup is-highlighted">
+        <col class="col col6">
+      </colgroup>
+      <thead>
+        <tr>
+          <th>Town Name</th>
+          <th>Founding feature</th>
+          <th>Weaknesses</th>
+          <th>Strengths</th>
+          <th>Primary Inhabitants</th>
+          <th>No. Inhabitants</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>Pernrith</th>
+          <td>Ancient water source</td>
+          <td>Low economy</td>
+          <td>Rare animal training and rare herbalism</td>
+          <td>People</td>
+          <td class="align-right">15</td>
+        </tr>
+        <tr>
+          <th>Fandralore</th>
+          <td>Strong metal ores</td>
+          <td>Armorsmithing</td>
+          <td>Sustainable hunting and prosperous trade</td>
+          <td>Elves</td>
+          <td class="align-right">20</td>
```

Comment:
NIT: Good thing to make a note of in the [Styleguide for Tables](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Tables)